conans/client/tools/oss.py

```diff
@@ -202,69 +202,69 @@ class _OSVERSIONINFOEXW(ctypes.Structure):
     def get_debian_version_name(version):
         if not version:
             return None
-        elif version.major() == "8.Y.Z":
+        elif version.major == 8:
             return "jessie"
-        elif version.major() == "7.Y.Z":
+        elif version.major == 7:
             return "wheezy"
-        elif version.major() == "6.Y.Z":
+        elif version.major == 6:
             return "squeeze"
-        elif version.major() == "5.Y.Z":
+        elif version.major == 5:
             return "lenny"
-        elif version.major() == "4.Y.Z":
+        elif version.major == 4:
             return "etch"
-        elif version.minor() == "3.1.Z":
+        elif version.minor == 1:
             return "sarge"
-        elif version.minor() == "3.0.Z":
+        elif version.minor == 0:
             return "woody"
 
     @staticmethod
     def get_win_version_name(version):
         if not version:
             return None
-        elif version.major() == "5.Y.Z":
+        elif version.major == 5:
             return "Windows XP"
-        elif version.minor() == "6.0.Z":
+        elif version.major == 6 and version.minor == 0:
             return "Windows Vista"
-        elif version.minor() == "6.1.Z":
+        elif version.major == 6 and version.minor == 1:
             return "Windows 7"
-        elif version.minor() == "6.2.Z":
+        elif version.major == 6 and version.minor == 2:
             return "Windows 8"
-        elif version.minor() == "6.3.Z":
+        elif version.major == 6 and version.minor == 3:
             return "Windows 8.1"
-        elif version.minor() == "10.0.Z":
+        elif version.major == 10:
             return "Windows 10"
```

Comment:
```suggestion
        elif version.major == 10:
            return "Windows 10"
        elif version.major == 11:
            return "Windows 11"
```
(or perhaps these suggestions do not belong in this pr)